Apps/WebClient/src/ClientApp/src/views/timeline.vue

```diff
@@ -351,7 +351,11 @@ export default class TimelineView extends Vue {
                         </span>
                     </b-alert>
                     <b-alert
-                        :show="immunizationNeedsInput"
+                        :show="
+                            immunizationNeedsInput &&
+                            (immunizationIsDeferred ||
```

Comment:
Instead of doing the logic here, I think it would be better if the immunizationNeedsInput would get updated if the immz count is zero

Reply:
I knew it :-)
Fixed now